### ISSUE TITLE
Upper/Lowercase keys on json body

### ISSUE THREAD

**Issue by aeosmanoglu** [pass-wall/passwall-server#35]
Your json returns have different keys. For example when we post a generate password request it returns:

```json
{
  "Status": "Success",
  "Message": "E$X+oVdgfBF=Whzm"
}
```

or

```json
{
  "code": 401,
  "message": "Token is expired"
}
```

So you create two different key `Message` and `message`. Please, can you review your code and standardize your keys? It's hard to parse json data.

**Comment by yakuter** [comment]
Well only auth jsons are different then api's responses because of the external package I use. I will work on this to make them same.

**Comment by yakuter** [comment]
Done! Now all the json responses are identical and snake_case
